Jenkinsfile

```diff
@@ -228,42 +207,34 @@ pipeline {
 
         //============================================ Deploy ==================================================================
 
-        stage('Release') {
+        stage('Release RC on NPM') {
             when {
                 beforeAgent true
                 allOf {
                     expression { BRANCH_NAME ==~ /(release)/ }
                     environment name: 'COMMIT_PARENTS_COUNT', value: '1'
+                    expression { params.RELEASE == false }
                 }
             }
-            parallel {
-                stage('Publish on NPM') {
-                    steps {
-                        script {
-                            executeNpmLogin()
-                            nodeCmd("NODE_ENV=\"production\" npm publish")
-                        }
+            steps {
+                    script {
+                        executeNpmLogin()
+                        nodeCmd("NODE_ENV=\"production\" npm publish --tag rc")
                     }
                 }
             }
-        }
-
-        stage('Release Beta') {
+        stage('Release Devel on NPM') {
             when {
                 beforeAgent true
                 allOf {
-                    expression { BRANCH_NAME ==~ /(beta)/ }
-                    environment name: 'COMMIT_PARENTS_COUNT', value: '1'
+                    expression { BRANCH_NAME ==~ /(devel)/ }
+                    environment name: 'COMMIT_PARENTS_COUNT', value: '2'
```

Comment:
Doesn't this check block possible squash commits in devel?

Reply:
I forgot about that! I'll look for a better way to detect merges

Reply:
after some trial and error, we concluded that it's probably better to publish the devel tag on any commit on the branch (it is push-protected anyway)